src/komponenter/delseksjon.tsx

```diff
@@ -177,29 +165,55 @@ export function Delseksjon({ delseksjon, delseksjonIndeks }: seksjonProps) {
         oppdaterAvsnitt(fritekstStreng);
         settFritekst(fritekstStreng);
 
+        const mellomlagringDelseksjonKopi = dypKopi(
+            mellomlagringDelseksjonerState[delseksjonIndeks]
+        );
+        mellomlagringDelseksjonKopi.innhold[innholdIndeks].valgVerdi = nyTekstOgIndeksStreng;
+
         //finner flettefelt referanser i valgt dropdown option og setter nye flettefelt i flettefeltDropdown state.
         if (erInnholdDropdown(delseksjon.innhold[innholdIndeks])) {
-            let flettefeltNy: SanityChildren[] = [];
-
-            (delseksjon.innhold[innholdIndeks] as SanityDropdown).valg[
+            const flettefeltNy = finnFlettefeltIDropdown(
+                delseksjon.innhold[innholdIndeks] as SanityDropdown,
                 optionValgIndeks
-            ].tekst.forEach((del) => {
-                flettefeltNy = [...flettefeltNy, ...finnFlettefelt(del)];
-            });
+            );
             oppdaterFlettefeltFraDropdowns(flettefeltNy, innholdIndeks);
+
+            mellomlagringDelseksjonKopi.innhold[innholdIndeks].flettefelt = Array(
+                flettefeltNy.length
+            ).fill('');
```

Comment:
Gjøres dette for å fylle flettetabellen med tomme strenger? F.eks. om den finner tre flettefelt så får man `["", "", ""]`?